.github/workflows/linux-package.yml

```diff
@@ -0,0 +1,78 @@
+name: Linuax Package
+on:
+  push:
+    tags:
+      - 'opentrack-[0-9]+.[0-9]+.[0-9]+'
+      - 'opentrack-[0-9]+.[0-9]+.[0-9]+-[a-zA-z0-9]+'
```

Comment:
For `-rc` tags.

Reply:
These are globs, you can't use `+` with its regex meaning.

Hell, just make it `opentrack-*` and leave it at that.

Reply:
Don't worry, it is fully compatible with the doc. https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet